.travis.yml

```diff
@@ -9,6 +9,21 @@ before_install:
 
 matrix:
   include:
+    - os: linux
+      dist: bionic
+      arch: ppc64le
+      compiler: g++
+      sudo: required
+      install:
+        - export VULKAN_SDK=$TRAVIS_BUILD_DIR/$VK_VERSION/ppc64le
+        - sudo apt-get -qq update
+        - sudo apt-get install -y make gcc g++ mesa-common-dev libglu1-mesa-dev libxxf86dga-dev libxxf86vm-dev libasound2-dev libx11-dev libxcb1-dev 
+        - curl -L -v -o vulkansdk-linux-ppc64le-$VK_VERSION.tar.gz -O https://raw.githubusercontent.com/runlevel5/vulkansdk-linux-ppc64le/master/vulkansdk-linux-ppc64le-$VK_VERSION.tar.gz 
```

Comment:
Instead of building all components of SDK on-the-fly, I decide to build a binary package and use it for the time-being. Let's hope the upstream Vulkan would get ppc64le binary in the next release.